libs/cdk/utils/services/rtl.service.ts

```diff
@@ -14,12 +15,17 @@ export class RtlService {
     /** RTL value */
     rtl: BehaviorSubject<boolean>;
 
+    /** Signal wrapper for RTL value. */
+    rtlSignal: Signal<boolean | undefined>;
+
     /** @hidden */
     constructor(@Optional() @Inject(RTL_LANGUAGE) injectedRtlLanguages: string[]) {
         injectedRtlLanguages = injectedRtlLanguages || DefaultRtlLanguages;
 
         const filtered = injectedRtlLanguages.filter((language) => navigator.language.includes(language));
 
         this.rtl = new BehaviorSubject(filtered.length > 0);
+
+        this.rtlSignal = toSignal(this.rtl);
```

Comment:
Why have `undefined` in type ? It's either rtl or not. If it is because of the toSignal signature, then it has a second parameter for options where you can request to sync and have type `Signal<boolean>`